scripts/uc-def/uc-def

```diff
@@ -47,10 +47,19 @@ def verify_access_flags(data):
 			raise Exception("'%c' flag of flags '%s' not valid" % (i, data))
 
 def next_useful_line(tag, strip_newline=True, strip_comment=True):
+	if not ('input_lines' in tag):
+		tag['input_lines'] = tag['input_file'].readlines()
```

Comment:
i dont want spend my time making bugreport and testsuit to python. 
short: on my winXP git has checkoput .ucf with linux \n lienends. looks like python string engine converts somehow internaly to windows \r\n lineneds. therefore tell() counts 1 more byte per string. after 20 strings tell() position  points after actual string begin about 20bytes.